python/examples/extract_satellite_info.py

```diff
@@ -44,21 +44,26 @@
             logger.info('Loading %s from log %s.' % (os.path.basename(input_path), log_id))
     except FileNotFoundError as e:
         logger.error(str(e))
-        os.exit(1)
+        sys.exit(1)
 
-    # Read pose data from the file.
+    # Read satellite data from the file.
     reader = FileReader(input_path)
     result = reader.read(message_types=[GNSSSatelliteMessage], show_progress=True)
+    satellite_data = result[GNSSSatelliteMessage.MESSAGE_TYPE]
+    if len(satellite_data.messages) == 0:
```

Comment:
Is it necessary to bail out here? It looked earlier we handle the case where there is no GNSS data

Reply:
I think you're thinking of a different script?

Reply:
You're right!